Assets/MixedRealityToolkit.Tests/EditModeTests/Core/Definitions/Devices/MixedRealityControllerMappingProfileTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.OpenVR.Input;
+using NUnit.Framework;
+using UnityEditor;
+
+public class MixedRealityControllerMappingProfileTests
+{
+    /// <summary>
+    /// Verifies that the TestMixedRealityControllerMappingProfile was successfully updated.
```

Comment:
Can you maybe elaborate slightly here why we are testing UpdateInteractionSettingsFromDefault?